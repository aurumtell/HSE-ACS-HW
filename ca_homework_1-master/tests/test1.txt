1 movie1 2031 dayana
2 movie2 1940 islam
1 movie3 1438 dfgdfgfd
3 movie4 1400 200000
2 movie5 1400 fdgdf
3 movie6 -3252 -24
3 movie7 sdfsdfs vcbcvbc
3 movie8 -325dfgdfg -2asdas4