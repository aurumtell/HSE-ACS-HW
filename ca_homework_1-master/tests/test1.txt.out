Input container:
Container contains 7 elements.
0: Fiction: name = movie1, year = 2031, director = dayana, year / name.len(): 338.5
1: Cartoon: name = movie2, year = 1991, type = DRAW, year / name.len(): 331.833
2: Fiction: name = movie3, year = 1985, director = dfgdfgfd, year / name.len(): 330.833
3: Documentary: name = movie4, year = 2020, duration = 200000, year / name.len(): 336.667
4: Cartoon: name = movie5, year = 1954, type = DRAW, year / name.len(): 325.667
5: Documentary: name = movie6, year = 1967, duration = 5, year / name.len(): 327.833
6: Documentary: name = movie7, year = 1980, duration = 9, year / name.len(): 330
